app/Http/Controllers/UserController.php

```diff
@@ -47,8 +51,29 @@ public function followUser(User $profileId)
 
     public function unFollowUser(User $profileId)
     {
-        
         $profileId->followers()->detach(auth()->user()->id);
         return redirect()->back()->with('success', 'Successfully unfollowed the user.');
     }
+
+    public function showCategories()
```

Comment:
I will merge this PR, but i have some comments. this function is not related to Users it is more appropriate that you have a separated controller for Categories and transfer this function and to that controller